econml/_ortho_learner.py

```diff
@@ -540,27 +556,22 @@ def fit(self, Y, T, X=None, W=None, Z=None, sample_weight=None, sample_var=None,
     def _fit_nuisances(self, Y, T, X=None, W=None, Z=None, sample_weight=None):
         # use a binary array to get stratified split in case of discrete treatment
         stratify = self._discrete_treatment or self._discrete_instrument
+        strata = self._strata(Y, T, X=X, W=W, Z=Z, sample_weight=sample_weight)
```

Comment:
It could be.  For example, if DML's fit method is called with bootstrap inference and sample_var is passed there, then it will also be passed through to _strata by the bootstrap code (which just gets a set of args and kwargs and passes them through directly).